test/unit/google-apis/auth-client.ts

```diff
@@ -56,11 +56,13 @@ function verifyReportedMessage(
       }
       logs.info += text;
     },
-  };
-  const config2 = new Configuration(config1, logger as Logger);
+  } as {} as Logger;
+  const config2 = new Configuration(config1, logger);
   // tslint:disable-next-line:no-unused-expression
   new RequestHandler(config2, logger);
-  assert.deepStrictEqual(logs, expectedLogs);
+  setImmediate(() => {
```

Comment:
@DominicKramer I didn't feel great about needing to add this, but as far as I can tell it was kinda weird that this code worked without it.